index.html

```diff
@@ -0,0 +1,702 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link
+rel="shortcut icon"
+      type="image/png"
+      href="./Images/shortcut-icon.png"
+    />
+ 
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TNW</title>
+  <link rel="stylesheet" href="./CSS/reset.css" type="text/css">
+  <link rel="stylesheet" href="./CSS/style.css" type="text/css">
+  <link rel="stylesheet" href="./CSS/media.css" type="text/css">
+  <link href="https://fonts.googleapis.com/css2?family=Nova+Square&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Rajdhani:wght@500&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/13a4f3443f.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="chair">
+    <i class="fas fa-wheelchair"></i>
+  </div>
+  <!-- <div class="flexbox-1"> -->
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/